cpp/src/dbscan/dbscan.cu

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include <cuML_api.h>
+#include <cuml_api.h>
```

Comment:
Does cuml has some specification around `#include <>` and `#include ""`?  Sometime compiler flags generated by CMake like `-isystem` and `-I` and help preventing wrong include file.

Reply:
That's a good question! My rule of thumb would be: if referring to a header file inside of `src` and/or `src_prims` folder, better to use double-quotes and use <> for the rest. However, I'm open to discussing this with others too: @cjnolet @dantegd ?

Reply:
I have the same idea as @teju85

Reply:
[Read more](https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html) about this. I think it is better to refer to *local* headers with double quotes and the rest with angle brackets.

For this purpose, I created a simple script `cpp/scripts/include_checker.py` which enforces a uniform '#include' syntax. Filed issue #1179 to clean up the rest of folders in cpp.